### DISCUSSION TITLE
LaunchBar action for media keys control

### DISCUSSION THREAD

**Discussion by andrewsardone** [Ptujec/LaunchBar#12] [Q&A]
Does anyone have a LaunchBar action that behaves like they play/pause and seek buttons (media keys) do on one's TouchBar or keyboard?

I have explicit actions for play/pausing in Spotify, _but_ I'm wondering if anyone's developed one that carries the same behavior as the media keys.

**Comment by Ptujec** [comment]
No idea. Sorry
